### ISSUE TITLE
Figure 4 

### ISSUE THREAD

**Issue by guo794** [HVision-NKU/SRFormer#24]
In the link you provided for calculating the power spectrum, the final image is not a smooth curve, what did you do to turn it into a figure as the Figure 4.

**Comment by Z-YuPeng** [comment]
Right, I use the spacy to smooth the curve. There are many methods you can use to acheive simailr smooth effect.

**Comment by guo794** [comment]
I get it. Thank you so much for your reply!

**Comment by Tong-Kai** [comment]
Hello, which part of the code is for drawing figure 4
